app/furniture/marketplace/cart_products_controller.rb

```diff
@@ -24,6 +24,7 @@ def create
             turbo_stream.replace("cart-product-#{cart_product.product_id}", cart_product),
             turbo_stream.replace("cart-footer-#{cart.id}",
               partial: "marketplace/carts/footer", locals: {cart: cart}),
+              turbo_stream.replace("cart-total-#{cart.id}", partial: "marketplace/carts/total", locals: {cart: cart})
```

Comment:
I notice we make this particular change 3 different times in this controller; what if we gathered these three replace calls into a method so they aren't duplicated everywhere? What arguments would the method need to take to work appropriately across it's different calls?